components/well/index.css

```diff
@@ -11,19 +11,18 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-Well {
-  --spectrum-well-padding: var(--spectrum-global-dimension-size-200);
-  --spectrum-well-border-width: var(--spectrum-alias-border-size-thin);
-  --spectrum-well-margin-top: var(--spectrum-global-dimension-size-50);
-  --spectrum-well-min-width: var(--spectrum-global-dimension-size-3000);
+  --spectrum-well-border-width: var(--spectrum-border-width-100);
 }
 
 .spectrum-Well {
   text-align: start;
   display: block;
   min-inline-size: var(--spectrum-well-min-width);
-  padding: var(--spectrum-well-padding);
-  margin-block-start: var(--spectrum-well-margin-top);
-  border-width: var(--spectrum-well-border-width);
+  padding: var(--mod-well-padding, var(--spectrum-well-padding));
+  margin-block-start: var(--mod-well-margin-top, var(--spectrum-well-margin-top));
+  border-width: var(--mod-well-border-width, var(--spectrum-well-border-width));
   border-style: solid;
-  border-radius: var(--spectrum-alias-border-radius-regular);
+  border-radius: var(--mod-well-border-radius, var(--spectrum-well-border-radius));
+  background-color: var(--mod-well-background-color, var(--spectrum-well-background-color));
+  border-color: var(--mod-well-border-color, var(--spectrum-well-border-color));
```

Comment:
Rather than keeping these background colors custom, could we maybe utilize `--spectrum-gray-800` for the background color? For border, if we still need to use custom tokens for light/dark, they should no longer reference tokens in the vars folder. Could you add the custom tokens to the `components/tokens/custom-spectrum` folder in the correct color theme file? For example, adding `  --spectrum-well-border-color: rgba(255,255,255,0.05);` to  `components/tokens/custom-spectrum/custom-dark-vars.css`